Charts/Classes/Renderers/PieChartRenderer.swift

```diff
@@ -554,17 +541,17 @@ public class PieChartRenderer: ChartDataRendererBase
                     
                     // draw the transparent-circle
                     context.beginPath()
-                    context.addEllipse(inRect: CGRect(
+                    context.addEllipse(in: CGRect(
                         x: center.x - secondHoleRadius,
                         y: center.y - secondHoleRadius,
                         width: secondHoleRadius * 2.0,
                         height: secondHoleRadius * 2.0))
-                    context.addEllipse(inRect: CGRect(
+                    context.addEllipse(in: CGRect(
                         x: center.x - holeRadius,
                         y: center.y - holeRadius,
                         width: holeRadius * 2.0,
                         height: holeRadius * 2.0))
-                    context.eoFillPath()
+                    context.fillPath()
```

Comment:
according to CGContextEOFillPath, the rule is even-odd fill rule. However fillPath() seems taking default rule which is winding. I suspect we need check this.